spec/session_spec.rb

```diff
@@ -41,4 +44,28 @@ class Session < Binance::Session
       expect(subject.timeout).to eq(nil)
     end
   end
+
+  context 'should init with proxy' do
```

Comment:
usar when: when proxy_url option is given

Reply:
el should no lo tengo que usar en el it??
```
context 'when ....' do
  it 'should ....' do

  end
end
```